### PR TITLE
Barleycorn

### PR DESCRIPTION
**Pull request by yiancar**
## Description

Adding Ginghams big brother:)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/barleycorn/barleycorn.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "barleycorn.h"
+
+void keyboard_pre_init_user(void) {
+  // Set our LED pins as output
+  setPinOutput(B5);
+  setPinOutput(C0);
+}
```
Comment (drashna) [inline keyboards/barleycorn/barleycorn.c:22]:
```suggestion
void keyboard_pre_init_kb(void) {
    // Set our LED pins as output
    setPinOutput(B5);
    setPinOutput(C0);
    keyboard_pre_init_user();
}
```

**Comment by yiancar** [comment]
@fauxpark #yourwishismycommand ?! anything else?

**Comment by yiancar** [comment]
I suppose we did great now?:)

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**